doc/source/whatsnew/v0.11.0.rst

```diff
@@ -105,27 +105,54 @@ Conversion
 
 Mixed Conversion
 
-.. ipython:: python
-   :okwarning:
+.. code-block:: ipython
 
-   df3['D'] = '1.'
-   df3['E'] = '1'
-   df3.convert_objects(convert_numeric=True).dtypes
+    In [12]: df3['D'] = '1.'
 
-   # same, but specific dtype conversion
-   df3['D'] = df3['D'].astype('float16')
-   df3['E'] = df3['E'].astype('int32')
-   df3.dtypes
+    In [13]: df3['E'] = '1'
+
+    In [14]: df3.convert_objects(convert_numeric=True).dtypes
+    Out[14]:
+    A    float32
+    B    float64
+    C    float64
+    D    float64
+    E      int64
+    dtype: object
+
+    # same, but specific dtype conversion
+    In [15]: df3['D'] = df3['D'].astype('float16')
```

Comment:
Maybe not a big issue since this matches current documentation, but are these types supposed to be exactly the same as the above? Comment suggests so but result does not

Reply:
I don't think they are supposed to be exactly the same. `convert_objects` did a generic conversion, so using the 64 bit version by default. With astype you could specify a specific type.